src/main/settings.py

```diff
@@ -222,11 +223,27 @@ def resource_spatialdimension():
     STORAGES |= STORAGE_AZURE #update storages with storage_azure
 
 # -----Queue
-AZURITE_QUEUE_CONNECTION_STRING = os.getenv("AZURITE_QUEUE_CONNECTION_STRING")
-QUEUE_ACCOUNT_URL = os.getenv("QUEUE_ACCOUNT_URL")
 JOB_QUEUE_NAME = "job-queue"
 IMPORT_DRY_RUN_FIRST_TIME = False
 
+if os.getenv("AZURE_FEDERATED_TOKEN_FILE"):
+    credentials = WorkloadIdentityCredential()
+    name_storage_account =  os.getenv("AZURE_STORAGE_ACCOUNT_NAME")
+    QUEUE_ACCOUNT_URL = f"https://{name_storage_account}.queue.core.windows.net"
+    QUEUE_CLIENT = QueueClient(
+            credential=credentials,
+            account_url= QUEUE_ACCOUNT_URL,
+            queue_name= JOB_QUEUE_NAME,
+        )  
+elif AZURITE_QUEUE_CONNECTION_STRING := os.getenv("AZURITE_QUEUE_CONNECTION_STRING"): # for local development     
+    queue_service_client = QueueServiceClient.from_connection_string(AZURITE_QUEUE_CONNECTION_STRING)
+    try: # create queue for the first time
+        queue_service_client.create_queue(JOB_QUEUE_NAME)
+    except:
+        pass    
+    QUEUE_CLIENT = queue_service_client.get_queue_client(JOB_QUEUE_NAME)
```

Comment:
Denk je dat er een `else` nodig is? Want als beide omgevingsvariabelen niet zijn gezet willen we misschien een fallback of een melding loggen?

Reply:
dit is alleen relevant voor lokaal ontwikkelen, met debug=true komt er dan informatie terug als niet gezet. dat lijkt me voldoende.